topics/contributing/tutorials/updating_tutorial/tutorial.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: tutorial_hands_on
+
+title: Updating tool versions in a tutorial
+subtopic: getting-started
+priority: 6
+
+questions:
+- How can I update the tool versions in a tutorial in the GTN?
+- What else do these updates impact, and how do I update that for consistency?
+
+objectives:
+- Implement tutorial tool version update on existing GTN material
+
+time_estimation: 1H
+key_points:
+- The GTN has a number of fantastic features that make it a cutting-edge resource.
+- Jumping into existing materials can be daunting, but by following these steps, you can be a contribute to this vibrant community!
+
+contributions:
+  authorship:
+    - nomadscientist
+  editing:
+    - wee-snufkin
+
+requirements:
+-
+  type: "internal"
+  topic_name: contributing
+  tutorials:
+      - github-interface-contribution
+
+---
+
+Here, we provide a clear set of instructions for updating a GTN tutorial that uses the Galaxy interface for performing analysis. The GTN, and Galaxy, have a number of features to make it as reproducible and shareable as possible, so navigating what needs to be done for an overall update may feel daunting - but not anymore!
+
+We encourage you to pick a tutorial to try this on, so you can use this tutorial side-by-side.
+
+# Phase 1: Find a tutorial with an outdated workflow
+
+{% include _includes/cyoa-choices.html option1="No" option2="Yes" default="No"
+       text="Do you already have a tutorial and workflow you want to update?" %}
+
+<div class="No" markdown="1">
+
+> <hands-on-title>Check for an outdated workflow</hands-on-title>
+>
+> 1. **Find a {% icon hands_on %} Hands-on tutorial** in your training topic of interest (for example, Single-cell)
```

Comment:
Have you tried to put the `div` directly in the hands-on box? As the steps are pretty similar except the first one, that could be great to have only 1 hands-on box with 2 different step 1. That will avoid the redundancy

Reply:
I'll try!

Reply:
Damn it